pages/webview/browse.py

```diff
@@ -1,45 +1,61 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
 from pypom import Region
-from selenium.webdriver.common.by import By
```

Comment:
This doesn't need to be changed but a lot of the imports are incorrect. The order of imports should be:
1. standard lib
2. libraries
3. application modules

Sooo. selenium should be above any pages that are imports. This doesn't need to be changed right now. We can do it later.

Reply:
a lot of that is my fault =(